README.md

```diff
@@ -314,9 +317,79 @@ All inputs are optional as they may be specified in the Atlas configuration file
    For example: `'{"var1": "value1", "var2": "value2"}'`.
 * `working-directory` - The working directory to run from.  Defaults to project root.
 
+### `ariga/atlas-action/schema/push`
+
+Push a schema version with an optional tag to Atlas.
+
+#### Inputs
+
+All inputs are optional as they may be specified in the Atlas configuration file.
+
+* `working-directory` - The working directory to run from.  Defaults to project root.
+* `config` - The URL of the Atlas configuration file.  By default, Atlas will look for a file
+  named `atlas.hcl` in the current directory. For example, `file://config/atlas.hcl`.
+  Learn more about [Atlas configuration files](https://atlasgo.io/atlas-schema/projects). 
+* `env` - The environment to use from the Atlas configuration file.  For example, `dev`.
+* `vars` - Stringify JSON object containing variables to be used inside the Atlas configuration file.
+  For example: `'{"var1": "value1", "var2": "value2"}'`.
+* `dev-url` - The URL of the dev-database to use for analysis. For example: `mysql://root:pass@localhost:3306/dev`.
+  Read more about [dev-databases](https://atlasgo.io/concepts/dev-database).
+* `schema-name` - The name (slug) of the project in Atlas Cloud.
+* `description` - The description of the schema changes.
+* `version` - The version of the schema changes.
+* `tag` - The tag to apply to the pushed schema. By default the current git commit hash is used.
```

Comment:
what's the difference between version and tag?

Reply:
The atlas push will generate new migration file with the give version. And pushed as the tag.